.travis.yml

```diff
@@ -41,7 +41,7 @@ before_install:
 install:
   # To be removed when this issue will be resolved: https://github.com/composer/composer/issues/5355
   - if [[ "$COMPOSER_FLAGS" == *"--prefer-lowest"* ]]; then composer update --prefer-dist --no-interaction --prefer-stable --quiet; fi
-  - composer update ${COMPOSER_FLAGS} --prefer-dist --no-interaction
+  - COMPOSER_MEMORY_LIMIT=-1 composer update ${COMPOSER_FLAGS} --prefer-dist --no-interaction
```

Comment:
In our open source bundles we are managing the setup of unlimited memory through the file `travis/z_php.ini` and adding this `phpenv config-add travis/z_php.ini` in the `before_install` section. In core bundle the file `z_php.ini` already exists. For this reason I would setup the memory limit of all the pipelines in the same way

Reply:
That increases PHP's memory globally... Which could potentially hide other problems down the line. I'm all in for consistency across pipelines, but I don't think we should follow that here.

Reply:
Ok , sounds a good reason for me. Can then delete the folder travis that contains just the file z_php.ini because was there just for this purpose?

Reply:
Gone